src/app/help/en/welcome.md

```diff
@@ -8,71 +9,65 @@ Below that is a section of "Helpful Posts from Steemit Users", which contains a
 
 Below that is a list of recommended users to follow, a collection of other resources including the <a href="/faq.html">FAQ Page</a>, and information on where to find live help. 
 
-<a class="anchor" name="Table_of_Contents"></a>
-## Table of Contents
-
+## <span id="Table_of_Contents">Table of Contents</span>
 ### Quick Start Guide
 
-- <a href="/welcome#No_Cost_to_Participate">No Cost to Participate</a>
-- <a href="/welcome#Upvotes">Upvotes</a>
-- <a href="/welcome#Comments">Comments</a>
-- <a href="/welcome#Creating_Posts">Creating Posts</a>
-- <a href="/welcome#Tags">Tags</a>
-- <a href="/welcome#Followers_and_Feeds">Followers and Feeds</a>
-- <a href="/welcome#Resteem">Resteem</a>
-- <a href="/welcome#Digital_Currencies">Digital Currencies</a>
-- <a href="/welcome#Curation">Curation</a>
-- <a href="/welcome#Payments">Payments</a>
-- <a href="/welcome#Home,_New,_Hot,_Trending,_Promoted,_and_Active">Home, New, Hot, Trending, Promoted, and Active</a>
-- <a href="/welcome#Profile">Profile</a>
-- <a href="/welcome#Reputation">Reputation</a>
-- <a href="/welcome#Cashing_out_or_Spending_SBD">Cashing out or Spending SBD</a>
-- <a href="/welcome#Plagiarism">Plagiarism</a>
-- <a href="/welcome#Password_Security">Password Security</a>
-- <a href="/welcome#Earning_on_Steemit">Earning on Steemit</a>
+- <a href="#No_Cost_to_Participate">No Cost to Participate</a>
```

Comment:
couldn't these be done with regular markdown links?

Reply:
Yes they could. I'm not sure why they were done with < a > tags.